polkadot-protocol/protocol-components/overview.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Overview
+description: Explore the fundamental concepts of the Polkadot protocol, including account addresses, extrinsics, events and runtime upgrades.
+---
+
+# Overview
+
+This page offers a comprehensive introduction to the Polkadot protocol, covering key terminology unique to Polkadot, significant distinctions from other blockchain networks, and practical guidance for interacting with the chain.
+
+!!!note 
+    If the information provided below doesn't fully address your questions about the Polkadot protocol, be sure to visit the [Polkadot Specification](https://spec.polkadot.network/id-polkadot-protocol){target=\_blank}.
+
+## Tokens
+
+- Token decimals:
+    - Polkadot (DOT): 10
+    - Kusama (KSM): 12
+- Base unit: "Planck"
+- Balance type: `u128`
+
+### Redomination
+
+Polkadot conducted a community poll, which ended on **27 July 2020** at block **888,888** to decide whether to redenominate the DOT token. 
+
+The stakeholders chose to redenominate the token, changing the value of 1 DOT from 1e12 plancks to 1e10 plancks. 
+
+Importantly, this did not affect the total number of base units (plancks) in the network, only how a single DOT is represented.
+
+The redenomination became effective 72 hours after transfers were enabled, occurring at block 1,248,326 on 21 August 2020 around 16:50 UTC.
```

Comment:
According to the wiki: https://wiki.polkadot.network/docs/learn/learn-redenomination, this redenomination thing has happened  at block 1_248_328

```suggestion
The redenomination became effective 72 hours after transfers were enabled, occurring at block 1,248,328 on 21 August 2020 around 16:50 UTC.
```